### ISSUE TITLE
Support for CLASS

### ISSUE THREAD

**Issue by Spinlogic** [eerimoq/asn1tools#148]
Any plans to support CLASSes any time soon?

Parsing with CLASS works OK. But compiling crashes.  
The protocol that I want to compile is rather simple one from 3GPP TS29.168 (all the 3GPP protocols are freely availble at [their site](https://www.3gpp.org)). 
My intention is to develop the core logic of an open source cell broadcast centre (CBC) for public warning. This is very low load telco node that can be done in Python.

I don't have deep knowledge of ASN.1, but if someone gives me a serious hint on where to start, I can try to contribute this part if it is possible to do so without having to get too deep into ITU-T X standards.

**Comment by eerimoq** [comment]
No, there are no plans on implementing support for CLASSes, unfortunately. Would be great if you can help out implementing it. Unfortunately I think the specs is your best friend in how to do so =)
